lib/virt_autotest/utils.pm

```diff
@@ -222,6 +222,8 @@ sub create_guest {
     my $maxvcpus = $guest->{maxvcpus} // $vcpus + 1;    # same as for memory, test functionality but don't waste resources
     my $extra_args = get_var("VIRTINSTALL_EXTRA_ARGS", "") . " " . get_var("VIRTINSTALL_EXTRA_ARGS_" . uc($name), "");
     my $linuxrc = $guest->{linuxrc};
+    if (script_run("test -d /var/lib/libvirt/images/") != 0) {
+        assert_script_run 'mkdir -p /var/lib/libvirt/images/'; }
```

Comment:
```suggestion
    script_run "[ -d /var/lib/libvirt/images ] || mkdir /var/lib/libvirt/images";    
```

Reply:
It should be OK, I think, next time I try to use in your way.